buku

```diff
@@ -735,8 +740,11 @@ class BukuDb:
         title_in : str, optional
             Title to add manually. Default is None.
         tags_in : str, optional
-            Comma-separated tags to add manually.
-            Must start and end with comma. Default is None.
+            Comma-separated tags to add manually, instead of fetching them. Default is None.
```

Comment:
> Must start and end with comma.

This is not a condition that is actually observed, even in the tests for this method :sweat_smile: